configuration.go

```diff
@@ -53,3 +55,27 @@ func (c *ConfigurationService) GetPluginConfig() map[string]interface{} {
 func (c *ConfigurationService) SavePluginConfig(config map[string]interface{}) error {
 	return normalizeAppErr(c.api.SavePluginConfig(config))
 }
+
+// CheckRequiredServerConfiguration checks if the server is configured according to
+// plugin requirements.
+//
+// Minimum server version: 5.2
+func (c *ConfigurationService) CheckRequiredServerConfiguration(req *model.Config) (bool, error) {
```

Comment:
Is this something we want to expose, or just leave internal for our use?

Reply:
Sorry, I don't understand what you are referring to. `CheckRequiredServerConfiguration` is exported because it's a handy way for plugin to checkj

Reply:
Sorry, I don't understand what you are referring to. `CheckRequiredServerConfiguration` is exported because it's a handy way for plugin to check `Manifest.RequiredConfig`. It's not used internally anywhere right now.